scripts/asm-installer/install_asm

```diff
@@ -1705,6 +1745,20 @@ install_secrets() {
     --from-file="${CA_CHAIN}"
 }
 
+init_private_ca() {
+  WORKLOAD_IDENTITY="$PROJECT_ID.svc.id.goog[istio-system/istiod-service-account]"
```

Comment:
Will "init_private_ca" work with Hub workload identity in the future?

Reply:
yes, but NOT for preview. HUB workload identity is something we will integrate with for GA